stacks/storageos/yaml/storageos-cluster.yaml

```diff
@@ -0,0 +1,26 @@
+---
+# Source: storageos-operator/templates/storageoscluster_cr.yaml
+
+
+apiVersion: storageos.com/v1
+kind: StorageOSCluster
+metadata:
+  name: storageos
+  namespace: storageos
+spec:
+  namespace: kube-system
```

Comment:
Should this also be in the `storageos` namespace?

Reply:
@paulanunda thanks for reviewing.  `kube-system` is preferred since it lets us mark the containers providing storage as `priorityClassName: system-cluster-critical`.  We don't want the storage to be removed from underneath any apps, so this makes them last to be evicted if a node is overloaded.

Reply:
Thanks for the info.